libs/sdk-ui-tests-e2e/README.md

```diff
@@ -45,16 +66,16 @@ Currently, our ECR images do not support ARM64 tags, so to run Docker things in
 This will make your docker use upstream images that do support ARM64 and thus your Mac will run them natively.
 Do not commit this change!
 
-### Run tests against wiremock backend with live Cypress controls (tests are importing their recorded mapping files)
+### Run tests against mock backend with live Cypress controls (tests are importing their recorded mapping files)
 
 Good for debugging the tooling when tests are passing with live backend but fail on CI.
 Runs Cypress, Scenarios, and Wiremock separately in Docker.
 Make sure you have set correct SDK_BACKEND environment variable (.env)
 
--   Start recorded backend `npm run local-dev-wiremock` (add `--verbose` param for debugging)
--   Start Scenarios `npm run run-scenarios-local`
+-   Run `rushx create-ref-workspace` to create reference workspace or make sure you have valid TEST_WORKSPACE_ID specified in the `.env` file.
```

Comment:
should back to old contend and check that if it work. Your new change is mentioning "run isolated record" without live cypress control